CHANGELOG.md

```diff
@@ -1,3 +1,141 @@
+
+* Adds support for `CREATE INDEX` commands without index name on citus tables
+
+* Adds support for `CREATE TABLE ... USING table_access_method` statements
+
+* Adds support for co-located/recurring sublinks in the target list
+
+* Adds support for local execution in `INSERT..SELECT` commands that perform
+  re-partitioning
+
+* Propagates `ALTER TABLE .. SET LOGGED/UNLOGGED` statements
+
+* Propagates `ALTER INDEX .. SET STATISTICS` statements
+
+* Propagates `ALTER SCHEMA RENAME` statements
+
+* Propagates `CREATE STATISTICS` statements across workers.
+
+* Propagates `DROP STATISTICS` statements across the workers
+
+* Propagates all types of `ALTER STATISTICS` statements.
+
+* Improves performance in some subquery pushdown cases
+
+* Removes `citus.binary_master_copy_format` GUC
+
+* Removes `citus.expire_cached_shards` GUC
+
+* Removes `citus.large_table_shard_count` GUC
+
+* Removes `citus.max_assign_task_batch_size` GUC
+
+* Removes `citus.max_running_tasks_per_node` GUC
+
+* Removes `citus.max_task_string_size` GUC
+
+* Removes `citus.max_tracked_tasks_per_node` GUC
+
+* Removes `citus.sslmode` GUC
+
+* Removes `citus.task_tracker_delay` GUC
+
+* Removes the word 'master' from Citus UDFs
+
+* Removes unused UDF `mark_tables_colocated`
+
+* Removes `upgrade_to_reference_table` UDF
+
+* Renames 'master' to 'distributed' for columns of `citus_dist_stat_activity`
+
+* Renames 'master' to 'distributed' for columns of `citus_worker_stat_activity`
+
+* Avoids establishing new connections until the previous ones are finalized
+
+* Avoids re-using connections for intermediate results
+
+* Chooses the default co-location group deterministically
+
+* Deletes transactions when removing a node
+
+* Drops support for `create_citus_local_table` in favor of
+  `citus_add_local_table_to_metadata`
+
+* Fixes `EXPLAIN ANALYZE` error when query returns no columns
+
+* Fixes a bug preventing `INSERT SELECT .. ON CONFLICT` with a constraint name
+  on local shards
+
+* Fixes a bug related to correctness of multiple joins
+
+* Fixes a bug that might cause self-deadlocks when COPY used in xact block
+
+* Fixes a bug with partitioned tables that block new partition creation due to
+  wrong constraint names on workers
+
+* Fixes a crash that occurs when distributing table after dropping foreign key
+
+* Fixes a crash when adding/dropping foreign keys from reference to local
+  tables added to metadata
+
+* Fixes an unexpected error when executing `CREATE TABLE IF NOT EXISTS
+  PARTITION OF` commands
+
+* Fixes deadlock issue for `CREATE INDEX` in single node
+
+* Fixes distributed deadlock detection being blocked by metadata sync
+
+* Fixes handling indexes when adding local table to metadata
+
+* Fixes `undistribute_table` when table has column with `GENERATED ALWAYS AS
+  STORED` expressions
+
+* Improves concurrent index creation failure message
+
+* Prevents empty placement creation in the coordinator when executing
+  `master_create_empty_shard()`
+
```

Comment:
Should we have this pr in changelog https://github.com/citusdata/citus/pull/4489 ?

(I explicitly didn't add a description for that as we are just exposing adding local tables to metadata)

Reply:
I think we should not as you suggest